### PR TITLE
feat: disable `fail-fast` for candidates job

### PR DESCRIPTION
**Pull request by innovaker**
Ensures equal opportunities for each candidate.

### CODE REVIEW

File: .github/workflows/containers.yml [hunk 1]
```diff
@@ -81,6 +81,7 @@ jobs:
     if: ${{ !startsWith(github.ref, 'refs/tags') }}
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```
Comment (petejohanson) [inline .github/workflows/containers.yml:84]: Can you explain this one in a bit more detail? Not sure the advantage of this.

Comment (innovaker) [reply .github/workflows/containers.yml:84]: Ah, true, I think I got ahead of myself!  `fail-fast` stops other all in-progress jobs from being cancelled when one fails.  It doesn't make sense in the current batch but will make sense soon.  I'll put this PR on ice until it does.

**Comment by innovaker** [comment]
This PR complements #57 and #58.

File: .github/workflows/containers.yml (hunk 1 above)
Comment (petejohanson) [reply .github/workflows/containers.yml:82]: Ok, with all the other inline cache and dev/build tweaks, this makes total sense. Thanks.

**Review by petejohanson — APPROVED** [review]

**Pull request merged**